src/components/Button/index.tsx

```diff
@@ -0,0 +1,10 @@
+// Styles
+import './Button.scss'
+
+const Button = (props: { children: React.ReactNode }) => {
```

Comment:
It's been awhile since I've done this but I think you can just do:

```suggestion
const Button: React.FunctionComponent<{}> = props => {
```

And if you're really fancy, I'm sure you can lookup all the valid attributes for a `<button>` element and swap out `{}` for that instead.

Reply:
Probably it is `React.HTMLAttributes<HTMLButtonElement>` but don't quote me on that